docs/feature_unicode.md

```diff
@@ -177,54 +210,47 @@ You can customize the keys used to trigger Unicode input for macOS, Linux and Wi
 
 ```c
-#define UNICODE_CYCLE_PERSIST false
+send_unicode_string("(ノಠ痊ಠ)ノ彡┻━┻");
 ```
 
-!> Using `UNICODE_SELECTED_MODES` means you don't have to initially set the input mode in `matrix_init_user()` (or a similar function); the Unicode system will do that for you on startup. This has the added benefit of avoiding unnecessary writes to EEPROM.
+Example uses include sending Unicode strings when a key is pressed, as described in [Macros](feature_macros.md).
 
-## `send_unicode_string()`
+### `send_unicode_hex_string()`
 
-This function is much like `send_string()` but allows you to input UTF-8 characters directly, and supports all code points (provided the selected input method also supports it). Make sure your `keymap.c` is formatted in UTF-8 encoding.
+Similar to `send_unicode_string()`, but the characters are represented by their Unicode code points, written in hexadecimal and separated by spaces. For example, the table flip above would be achieved with:
 
 ```c
-send_unicode_string("(ノಠ痊ಠ)ノ彡┻━┻");
+send_unicode_hex_string("0028 30CE 0CA0 75CA 0CA0 0029 30CE 5F61 253B 2501 253B");
 ```
 
-## `send_unicode_hex_string()`
+An easy way to convert your Unicode string to this format is to use [this site](https://r12a.github.io/app-conversion/) and take the result in the "Hex/UTF-32" section.
 
-Similar to `send_unicode_string()`, but the characters are represented by their code point values in ASCII, separated by spaces. For example, the table flip above would be achieved with:
 
-```c
-send_unicode_hex_string("0028 30CE 0CA0 75CA 0CA0 0029 30CE 5F61 253B 2501 253B");
-```
+## Additional Language Support
 
-An easy way to convert your Unicode string to this format is by using [this site](https://r12a.github.io/app-conversion/), and taking the result in the "Hex/UTF-32" section.
+In `quantum/keymap_extras`, you'll see various language files — these work the same way as the ones for alternative layouts such as Colemak or BÉPO. When you include one of these language headers, you gain access to keycodes specific to that language / national layout. Such keycodes are defined by a 2-letter country/language code, followed by an underscore and a 4-letter abbreviation of the character to which the key corresponds. For example, including `keymap_french.h` and using `FR_UGRV` in your keymap will output `ù` when typed on a system with a native French AZERTY layout.
 
-## Additional Language Support
+If the primary system layout you use on your machine is different from US ANSI, using these language-specific keycodes can help your QMK keymaps better match what will actually be output on the screen. However, keep in mind that these keycodes are just aliases for the corresponding default US keycodes under the hood, and that the HID protocol used by keyboards is itself inherently based on US ANSI.
```

Comment:
I'm struggling to offer good advice for this section. I actually followed your doc to enable Unicode Map on my KC60, but I haven't yet worked out the magic solution to get this to work fully, because I mainly use English Dvorak at the OS level, and something about that seems to break the feature. It wasn't clear to me from this that it *only* works with US ANSI arrangement.

Enabling this to work with other keys in `unicode_input_start()` would be a good improvement for the future, but that's out of scope for this PR.